CP02PMCI/CP02PMCI_R00002_ingest.csv

```diff
@@ -1,11 +1,11 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/imm/adcpt/adcpt*.DAT,CP02PMCI-RII01-02-ADCPTG010,recovered_host,Not Expected,not recovered
-#mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/ADCP*/*_Recovered_Data_RDI_000.000,CP02PMCI-RII01-02-ADCPTG010,recovered_inst,Not Expected,not recovered
-#mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/irid2shore/stc_status.*_*.txt,CP02PMCI-SBS01-00-STCENG000,recovered_host,Not Expected,not recovered
-#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/3dmgx3/*_*.3dmgx3.log,CP02PMCI-SBS01-01-MOPAK0000,recovered_host,Not Expected,not recovered
-#mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/CWFP*/E*.DA?,CP02PMCI-WFP01-00-WFPENG000,recovered_wfp,Not Expected,not recovered
-#mi.dataset.driver.vel3d_k.wfp.vel3d_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/CWFP*/A*.DA?,CP02PMCI-WFP01-01-VEL3DK000,recovered_wfp,Not Expected,not recovered
-#mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/CWFP*/C*.DA?,CP02PMCI-WFP01-02-DOFSTK000,recovered_wfp,Not Expected,not recovered
-#mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/CWFP*/C*.DA?,CP02PMCI-WFP01-03-CTDPFK000,recovered_wfp,Not Expected,not recovered
-#mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/CWFP*/E*.DA?,CP02PMCI-WFP01-04-FLORTK000,recovered_wfp,Not Expected,not recovered
-#mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/CWFP*/E*.DA?,CP02PMCI-WFP01-05-PARADK000,recovered_wfp,Not Expected,not recovered
\ No newline at end of file
+#mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/imm/adcpt/adcpt*.DAT,CP02PMCI-RII01-02-ADCPTG010,recovered_host,Not Expected,ADCP Indcutive comms failed
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/CP02PMCI/R00002/cg_data/dcl00/ADCP*/*.000,CP02PMCI-RII01-02-ADCPTG010,recovered_inst,Available,not recovered

```

Comment:
The Raw Data Repo has 2 files that end with .000  (CP02PMCI-00002_ADCP_20195_RDI_000.000 and _RDI_000.000).  They are the same size, and presumably duplicates.